packages/opentelemetry-sdk-trace-node/test/NodeTracerProvider.test.ts

```diff
@@ -108,7 +108,6 @@ describe('NodeTracerProvider', () => {
         sampler: new AlwaysOnSampler(),
       });
       const span = provider.getTracer('default').startSpan('my-span');
-      assert.ok(span instanceof Span);
```

Comment:
note for reviewer: `Span` is not a class anymore but a type. If necessary we could check that conforms to the `api.Span` & `ReadableSpan` interfaces.

Reply:
it's fine to leave it out, I think.